wpiutil/src/main/native/cpp/timestamp.cpp

```diff
@@ -131,36 +186,32 @@ void wpi::SetNowImpl(uint64_t (*func)(void)) {
 uint64_t wpi::Now() {
 #ifdef __FRC_ROBORIO__
   // Same code as HAL_GetFPGATime()
-  if (!global) {
+  if (!hmb) {
     std::fputs(
         "FPGA not yet configured in wpi::Now(). Time will not be correct",
         stderr);
     std::fflush(stderr);
     return 0;
   }
-  int32_t status = 0;
-  uint64_t upper1 = global->readLocalTimeUpper(&status);
-  uint32_t lower = global->readLocalTime(&status);
-  uint64_t upper2 = global->readLocalTimeUpper(&status);
-  if (status != 0) {
-    goto err;
-  }
+
+  asm("dmb");
+  uint64_t upper1 = hmb.hmbBuffer[241];
```

Comment:
Add constants for these magic numbers (just `#define` or `static constexpr int ...` in this file would work).